src/app/shared/dso-selector/modal-wrappers/export-metadata-selector/export-metadata-selector.component.ts

```diff
@@ -82,24 +85,30 @@ export class ExportMetadataSelectorComponent extends DSOSelectorModalWrapperComp
     const parameterValues: ProcessParameter[] = [
       Object.assign(new ProcessParameter(), { name: '-i', value: dso.uuid }),
     ];
-    return this.scriptDataService.invoke(METADATA_EXPORT_SCRIPT_NAME, parameterValues, [])
-      .pipe(
-        getFirstCompletedRemoteData(),
-        map((rd: RemoteData<Process>) => {
-          if (rd.hasSucceeded) {
-            const title = this.translationService.get('process.new.notification.success.title');
-            const content = this.translationService.get('process.new.notification.success.content');
-            this.notificationsService.success(title, content);
-            if (isNotEmpty(rd.payload)) {
-              this.router.navigateByUrl(getProcessDetailRoute(rd.payload.processId));
-            }
-            return true;
-          } else {
-            const title = this.translationService.get('process.new.notification.error.title');
-            const content = this.translationService.get('process.new.notification.error.content');
-            this.notificationsService.error(title, content);
-            return false;
+    return this.authorizationDataService.isAuthorized(FeatureID.AdministratorOf).pipe(
+      switchMap((isAdmin) => {
+        if (isAdmin) {
+          parameterValues.push(Object.assign(new ProcessParameter(), {name: '-a'}));
+        }
+        console.log(isAdmin, parameterValues);
```

Comment:
This appears to be for debugging? Probably should be removed

Reply:
I removed the stray console log